File: work/spec/weather_spec.rb
require 'weather'

describe Weather do
  it 'exists!' do
    expect(subject.class).to eq Weather
  end

  it { is_expected.to respond_to(:weather) }

  describe '#weather' do  
    it 'weather to be stormy or not' do
      expect(subject.weather).to be(true).or be(false)
    end 
  end
end

File: jasmine/spec/weatherSpec.js
'use strict';

describe('Weather',function(){
  var weather;
  beforeEach(function(){
    weather = new Weather();
  });
  it('gives stormy sometimes', function(){
    spyOn(Math,'random').and.returnValue(1);
    expect(weather.isStormy()).toBeTruthy();
  });
  it('gives not stormy other times', function(){
    spyOn(Math,'random').and.returnValue(0);
    expect(weather.isStormy()).toBeFalsy();
  });
});
